main_test.go

```diff
@@ -175,6 +175,31 @@ func (ts TestStore) GetSession(name string) (*vaulted.Session, string, error) {
 	return s, ts.Passwords[name], nil
 }
 
+func (ts TestStore) CreateSession(name string) (*vaulted.Session, string, error) {
+	if !ts.VaultExists(name) {
+		return nil, "", os.ErrNotExist
+	}
+
+	s := &vaulted.Session{
+		Expiration: time.Unix(1136239446, 0),
+		Vars:       make(map[string]string),
+		SSHKeys:    make(map[string]string),
+	}
+	vault := ts.Vaults[name]
+
+	s.Name = name
+
+	for key, value := range vault.Vars {
+		s.Vars[key] = value
+	}
+
+	for key, value := range vault.SSHKeys {
+		s.SSHKeys[key] = value
+	}
+	os.Setenv("GOPATH", "/vaulted")
```

Comment:
This is dirtying the current process' environment, we should either stop that or put it back after we're done!

Reply:
Additionally, I'm not sure we should be setting this for all `CreateSession()` calls on the test store. We can probably have the test in question create a session (for `GetSession()` to return) and then modify the vault to have the additional env var so when a new session is generated, the var is included.